shared-libs/transitions/src/transitions/multi_report_alerts.js

```diff
@@ -362,17 +362,26 @@ const onMatch = change => {
 };
 
 module.exports = {
-  filter: (doc, info = {}) =>
-    !!(
+  name: TRANSITION_NAME,
+  _lineage: lineage,
```

Comment:
let's keep this at the bottom. It's only exported for testing purposes (the unspoken rule of _ prefixed export properties...).

Reply:
Done! So, if we want to export a function for testing but it's not intended to be used externally, then we prefix it with underscore 👍

Reply:
Well ... not really :) 
This _ prefix was done before we were using rewire. I greatly prefer rewire for testing non-exported properties and functions and keep the export clean.